README.md

```diff
@@ -112,16 +112,62 @@ Then, run the installation and remove the `.deb` file:
 ```bash
 dpkg -i /tmp/scw.deb && rm -f /tmp/scw.deb
 ```
+-->
 
 ## With a Docker Image
 
-For each release, we deliver a tagged image on the [Scaleway Docker Hub](https://hub.docker.com/r/scaleway/cli/tags) so can run `scw` in a sandboxed way: _Coming soon..._
+### Official releases (Coming soon..)
+
+<!-- For each release, we deliver a tagged image on the [Scaleway Docker Hub](https://hub.docker.com/r/scaleway/cli/tags) so can run `scw` in a sandboxed way: _Coming soon..._
 
 ```sh
 docker run scaleway/cli version
 ```
 -->
 
+### Manual installation
+
+You can build the `scw` CLI with Docker. If you have Docker installed, you can run:
+```sh
+docker build -t scaleway/cli .
+```
+
+You can then use the CLI as you would run any image:
+```sh
+docker run -i --rm scaleway/cli
+```
+
+In order to use it you must mount the scaleway configuration file:
+```sh
+docker run -i --rm -v $HOME/.config/scw/config.yaml:/.config/scw/config.yaml:ro scaleway/cli
+```
+
+If you want to use `scw` instead of `docker run` you can add the following in your `~/.bashrc`:
+```bash
+scw() {
+    docker run -i --rm -v $HOME/.config/scw/config.yaml:/.config/scw/config.yaml:ro scaleway/cli "$@"
+}
+export -f scw
+```
+
+Or if you use ZSH, add the following in your `~/.zshrc`:
+```zsh
+scw() {
+    docker run -i --rm -v $HOME/.config/scw/config.yaml:/.config/scw/config.yaml:ro scaleway/cli $@
+}
+```
+
+You can still use autocompletion while running the CLI in Docker, you just need to add the needed command after the definition of the new `scw` function.
+For instance, your `~/.bashrc` will look like this:
+```bash
+scw() {
+    docker run -i --rm -v $HOME/.config/scw/config.yaml:/.config/scw/config.yaml:ro scaleway/cli "$@"
```

Comment:
TODO: check if withiut `-t` we do not have probleme with terminal size detection

Reply:
done